frontend/src/app/pages/marker-page/marker-page.component.ts

```diff
@@ -22,6 +22,7 @@ import {Location} from '@angular/common';
 export class MarkerPageComponent implements OnInit {
 
     private static readonly SLICE_BATCH_SIZE = 10;
+    downloadingScanInProgress: boolean;
```

Comment:
I think its better to have default `false` value for this flag, especially when we are using it to render some new DOM elements (ngIf).